python/lsst/obs/subaru/crosstalk.py

```diff
@@ -511,16 +513,16 @@ def fixCcd(butler, visit, ccd, coeffs, display=True):
     """Apply cross-talk correction to a CCD, given the cross-talk coefficients"""
     mi = readImage(butler, visit=visit, ccd=ccd)
     if display:
-        ds9.mtv(mi.getImage(), frame=0, title="CCD %d" % ccd)
+        frame1 = afwDisplay.getDisplay(backend="ds9", frame=1)
```

Comment:
Is it necessary to hard-code the `ds9` backend here? It should be the default unless the user has overridden it -- and if they have, we presumably want to respect their choices unless this code actually requires `ds9`.